docs/src/markdown/contributing.md

```diff
@@ -57,7 +57,7 @@ There are a couple of dependencies that must be present before running the tests
 
 1. As ST3 is the only current, actively supported version, Python 3.3 must be used to validate the tests.
 
-2. Unit tests are run with `pytest`.  You can install pytest via:
+2. Unit tests are run with `pytest`.  You can install [pytest](https://pytest.org) via:
```

Comment:
I actually have a snippet that contains links: https://github.com/facelessuser/RegReplace/blob/master/docs/src/markdown/_snippets/links.md.  I prefer links like this to be placed there and then just do something like `[pytest][pytest]`.  That way we can reuse the link as an easy to remember key if desired.

Reply:
I noticed you used reference style links in other places, but I couldn't find their definitions at the end of the doc, so I just pasted the links in place. I'll keep in mind this for the future.